go.mod

```diff
@@ -122,6 +122,7 @@ require (
 )
 
 replace (
+	github.com/identitatem/dex-operator => github.com/leena-jawale/dex-operator v0.0.4-0.20220120230538-4b0d7539d854
```

Comment:
Keeping it for this PR. Will update go.mod for this in the next PR. Not sure how you people handle this. Do you first commit api changes and then make code changes?

Reply:
I'm admittedly not anywhere near expert on go modules, but I'm not sure why you'd need this in here at all. It seems like it's saying to use your copy of the code rather than what's in the local repo itself. What breaks if you just take this out?

Reply:
This PR is not yet merged. So api changes are not in local copy ‘identitatem/dex-operator’. Api changes are still in my forked repo branch. I was thinking we can merge this PR and after merge will open another PR to remove this line and update go.mod..Another option is to make separate PR for api change. Once that merge I’ll make another PR for code change. Which option sounds viable to you?